### ISSUE TITLE
unexpected symbol near '<'

### ISSUE THREAD

**Issue by aygupt1822** [fullylegit/ja3#8]
When I try to add the ja3.lua and md5.lua to the Wireshark's plugin folder in my Ubuntu machine I get the following errors :-

`Lua: syntax error: /home/ayush/.local/lib/wireshark/plugins/ja3.lua:8: unexpected symbol near '<'`

`Lua: syntax error: /home/ayush/.local/lib/wireshark/plugins/md5.lua:8: unexpected symbol near '<'`

Can someone help me with this issue ?
I am using Ubuntu 20.04.4 LTS

Surprisingly I, didnt got this issue in my Windows machine

**Comment by sergevanhaag** [comment]
@aygupt1822 Yeah, you probably downloaded the HTML page from `md5.lua`, but you meant to download the raw lua-file. So, grab this URL instead: `https://raw.githubusercontent.com/kikito/md5.lua/master/md5.lua`.
